### ISSUE TITLE
cannot generate pdf with styles option in real-time example

### ISSUE THREAD

**Issue by bboydflo** [PDFTron/web-to-pdf#1]
**Describe the bug**
I cannot generate a pdf when I add styles option.

This bug can be reproduced using one of the examples provided (real-time example)

**To Reproduce**
paste this code into examples/real-time/options.js

```js
module.exports = () => {
  const footer = `
    <div class="footer">
      <span>span 1</span>
      <span class="class1">span 2</span>
      <span class="class2">span3 | {{pageNumber}}</span>
    </div>
  `;

  const html = () => {
    return `
      <div class="Split">
        <p>Hello Page 1</p>
      </div>
      <div class="Split">
        <p>Hello Page 2</p>
      </div>
    `
  }

  return {
    templateSource: html,
    outputName: 'real-time',
    pageClass: 'Split',
    chunks: {
      footer
    }
  };
}
```

run `npx babel-node examples/real-time/index.js` => works fine!

add styles option

```
...
styles: [
  `
   .Split {
     color: red;
	 
     p { color: green; }
   }
  `
]
...
```

this also works fine!

close the current task and then try again with styles option => error!

in my case I get this error:

```
(node:2927) UnhandledPromiseRejectionWarning: Error: no parsers registered for: "
    "
    at parse (/Users/bboydflo/dev/personal/web-to-pdf/node_modules/snapdragon/lib/parser.js:473:15)
    at Parser.parse (/Users/bboydflo/dev/personal/web-to-pdf/node_modules/snapdragon/lib/parser.js:477:24)
    at Snapdragon.parse (/Users/bboydflo/dev/personal/web-to-pdf/node_modules/snapdragon/index.js:122:28)
    at Snapdragon.<anonymous> (/Users/bboydflo/dev/personal/web-to-pdf/node_modules/micromatch/lib/utils.js:47:45)
    at parse (/Users/bboydflo/dev/personal/web-to-pdf/node_modules/micromatch/index.js:739:26)    at memoize (/Users/bboydflo/dev/personal/web-to-pdf/node_modules/micromatch/index.js:859:13)
    at Function.micromatch.parse (/Users/bboydflo/dev/personal/web-to-pdf/node_modules/micromatch/index.js:745:10)
    at create (/Users/bboydflo/dev/personal/web-to-pdf/node_modules/micromatch/index.js:686:44)
    at /Users/bboydflo/dev/personal/web-to-pdf/node_modules/micromatch/index.js:695:16
    at memoize (/Users/bboydflo/dev/personal/web-to-pdf/node_modules/micromatch/index.js:859:13)
(node:2927) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handle
d with .catch(). (rejection id: 1)
(node:2927) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Expected behavior**
I'm want to be able to generate a report with above options.

**Desktop (please complete the following information):**
 - OS: macOS Catalina
 - Browser: Firefox Developer Edition

**Additional context**
In summary: I am able to generate a pdf if I don't add styles from the beginning but it fails if I do.
